lib/layer.js

```diff
@@ -15,6 +15,18 @@
 var pathRegexp = require('path-to-regexp')
 var debug = require('debug')('router:layer')
 
+/**
+ * Diagnostic channels
+ * @private
+ */
+var onHandleRequest
+var onHandleError
+try {
+  var dc = require('diagnostics_channel')
+  onHandleRequest = dc.channel('express.layer.handle_request')
+  onHandleError = dc.channel('express.layer.handle_error')
```

Comment:
These are probably not named correctly; I assume it should be `router.` prefix vs Express.

Reply:
Ah, yeah. Just copy-pasted from the other PR. I'll update those. :)